ffi-ctests/ctests/tests.c

```diff
@@ -0,0 +1,270 @@
+#include <stdlib.h>
+#include <stdbool.h>
+#include <string.h>
+
+#include <wirefilter.h>
+
+#define WIREFILTER_STRING(v, s) {(v).data = (const unsigned char *)(s); (v).length = strlen(s);}
```

Comment:
Instead of defining a variable with a macro, this could be a regular function that returns wire filter string as a result.